tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/FiniteDifference/Test_BoundaryConditionGhostData.cpp

```diff
@@ -721,21 +722,32 @@ SPECTRE_TEST_CASE(
   pypp::SetupLocalPythonEnvironment local_python_env{
       "PointwiseFunctions/AnalyticSolutions/"};
 
+  const std::array<double, 3> mean_velocity = {{0.23, 0.01, 0.31}};
```

Comment:
These options for SmoothFlow solution looks same as those used in `test()`. They can be moved into a global scope within this file. i.e. line 79